### DISCUSSION TITLE
While I import SimpleStorage contract and create a array of datatype SimpleStorage .

### DISCUSSION THREAD

**Discussion by vasanthmarimuth** [Cyfrin/foundry-full-course-cu#2183] [Q&A] [answered]
#code
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.10;

import "./SimpleStorage.sol";

contract StorageFactory{
    SimpleStorage[] public simpleStorageArray;
    
    function createSimpleStorageContract()public{
        SimpleStorage simpleStorage=new SimpleStorage();
        simpleStorageArray.push(simpleStorage);
    }
}
#Error
DeclarationError: Identifier not found or not unique.
 --> contracts/StorageFactory.sol:7:5:
  |
7 |     SimpleStorage[] public simpleStorageArray;
  |     ^^^^^^^^^^^^^

simpleStorageArraysimpleStorageArray

**Comment by EngrPips** [comment]
Hello @vasanthmarimuth, Please show your `SimpleStorage.sol` file

**Reply by vasanthmarimuth** [reply, marked as answer]
Thanks for your kindly reply @EngrPips  I finally correct it by my own . The mistake is that I make a contract name mistake in SimpleStorage file , Finally I finish and upload in my repository [ link:https://github.com/vasanthmarimuth/Interacting_with_other_contract].thank you for your kindly response.

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
